src/containers/Booksform.js

```diff
@@ -1,24 +1,64 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react';
+import { createBook } from '../actions/index';
+import { connect } from "react-redux";
+
+class BooksForm extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      title: '',
+      category: ''
+    }
+  }
+
+  handleChange(e) {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  handleSubmit(e) {
+    let { title, category } = this.state;
+    if (title.length !== 0 && category.length !== 0) {
+      this.props.saveBook(this.state);
+    } else {
+      alert('YOU MUST SUBMIT "TITLE" + "CATEGORY" TO SAVE');
+    }
+
+    this.setState({
+      title: '',
+      category: ''
+    });
+    e.preventDefault();
+  };
 
-class Booksform extends Component {  
   render() {
+    const { title, category } = this.state;
     const CATEGORIES = ["Action", "Biography", "History", "Horror", "Kids", "Learning", "Sci-Fi"]
     return (
       <form>
-        <input type="text" value={""} name="title"/>
-        <select value={''} name="category">
+        <input type="text" value={title} name="title" onChange={this.handleChange.bind(this)} />
+        <select value={(category) ? 'Action' : category} name="category" onChange={this.handleChange.bind(this)}>
+          <option value='default'>.... </option>
```

Comment:
This is not needed. A better experience would be to set the first category as the default in state and bind that to the option value. and after submits also reset the category to 'Action'. also add a placeholder for the input field.